RELEASE-NOTES.md

```diff
@@ -13,7 +13,7 @@
 - Add ability for deployment create and deployment update to create work pool queues — https://github.com/PrefectHQ/prefect/pull/8129
 
 ### Documentation
-- Add recipe contribution page and new recipe — https://github.com/PrefectHQ/prefect/pull/8183
+- Add recipe contribution page and [AWS Chalice](https://github.com/PrefectHQ/prefect-recipes/tree/main/flows-advanced/serverless) recipe — https://github.com/PrefectHQ/prefect/pull/8183
```

Comment:
I would personally link to [the recipe catalog in the docs](https://docs.prefect.io/recipes/recipes/#recipe-catalog) since the recipe itself isn't new to the Recipes repo. I would also link to the new recipe contribution page!